package.json

```diff
@@ -15,7 +15,7 @@
   "bugs": {
     "url": "https://github.com/ImUrX/vscode-scarpet/issues"
   },
-  "version": "0.1.8",
+  "version": "0.1.6",
```

Comment:
why is this in the past, i recommend force-pushing to the second commit you did

Reply:
I thought I had changed that...

Reply:
done